spring-xsuaa-it/src/main/java/testservice/api/basic/TestController.java

```diff
@@ -1,77 +0,0 @@
-/**
- * SPDX-FileCopyrightText: 2018-2022 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package testservice.api.basic;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.not;
-import static org.hamcrest.core.IsCollectionContaining.hasItem;
-import static org.junit.Assert.assertThat;
-
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.Profile;
-import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.annotation.AuthenticationPrincipal;
-import org.springframework.security.core.authority.SimpleGrantedAuthority;
-import org.springframework.security.oauth2.jwt.Jwt;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.client.RestTemplate;
-
-import com.sap.cloud.security.config.ClientCredentials;
-import com.sap.cloud.security.xsuaa.XsuaaServiceConfiguration;
-import com.sap.cloud.security.xsuaa.client.OAuth2TokenResponse;
-import com.sap.cloud.security.xsuaa.client.XsuaaDefaultEndpoints;
-import com.sap.cloud.security.xsuaa.client.XsuaaOAuth2TokenService;
-import com.sap.cloud.security.xsuaa.token.Token;
-import com.sap.cloud.security.xsuaa.tokenflows.ClientCredentialsTokenFlow;
-import com.sap.cloud.security.xsuaa.tokenflows.TokenFlowException;
-import com.sap.cloud.security.xsuaa.tokenflows.XsuaaTokenFlows;
-
-@RestController
-@Profile({ "test.api.basic" })
-public class TestController {
```

Comment:
have you replaced this test somewhere?

Reply:
Integration via Basic Auth is not tested anymore because TokenBrokerResolver will be moved out into sample code only.